website/docs/api/admin/feature-toggles-api.md

```diff
@@ -203,7 +235,13 @@ If the tuple (type, value) does not already exist, it will be added to the list
 
     - Returns _404-NOT-FOUND_ if the `type` was not found
 
-### Remove a tag from a Feature Toggle {#remove-a-tag-from-a-feature-toggle}
+## Remove a tag from a Feature Toggle {#remove-a-tag-from-a-feature-toggle}
+
+:::caution Deprecation notice
+<!-- Same as the above: how do we work with tags today? -->
+This endpoint is deprecated. Please use the [project-based endpoint to remove tags from a toggle](./feature-toggles-api-v2.md#fetching-toggles) instead.
```

Comment:
Where do you update tags in the new API?

Reply:
I don't think there are new endpoints for tags. 

But these tags endpoints are not under `disableLegacyFeaturesApi` either, so maybe they aren't deprecated?

Reply:
https://github.com/Unleash/unleash/blob/main/src/lib/routes/admin-api/feature.ts#L46-L61 :eyes: